ai2thor/controller.py

```diff
@@ -1335,8 +1352,13 @@ def start(
             unity_params = self.server.unity_params()
             self._start_unity_thread(env, width, height, unity_params, image_name)
 
-        # receive the first request
-        self.last_event = self.server.receive()
+        # receive the first request, give unity at least 5 minutes to start
+        self.last_event = self.server.receive(
+            timeout=max(
+                60.0 * 5,
+                self.server_timeout * 5 if self.server_timeout is not None else 0.0
```

Comment:
Why is this multiplied by `5` wouldn't it be better for the value to be the actual `timeout_value` so there were not unexpected behaviors?

Reply:
Decided to just create a `server_start_timeout` parameter. I was trying to keep the number of initialization parameters low but this was probably unnecessary.

> Also seems like the if condition will never be true because there is an assert for self.server_timeout is None in the constructor
The assert you're talking about checks that `self.server_timeout is None` or that it's positive.